src/server/modules/user/index.ts

```diff
@@ -49,7 +55,10 @@ export type GoUploadedFile = {
 }
 
 export type UpdateUrlOptions = Partial<
-  Pick<StorableUrl, 'state' | 'longUrl' | 'description' | 'contactEmail'> & {
+  Pick<
+    StorableUrl,
+    'state' | 'longUrl' | 'description' | 'contactEmail' | `tags`
```

Comment:
tiny nit, can we standardise on using the single quotation mark here?

Reply:
yes, will do

Reply:
interestingly, my IDE doesn't highlight this as good as Github

Reply:
done